cmd/prysmctl/withdrawal/withdrawal.go

```diff
@@ -0,0 +1,165 @@
+package withdrawal
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v3/beacon-chain/rpc/apimiddleware"
+	"github.com/prysmaticlabs/prysm/v3/io/prompt"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+	"go.opencensus.io/trace"
+)
+
+const (
+	basePath = "/eth/v1"
+	apiPath  = "/beacon/pool/bls_to_execution_changes"
+)
+
+func setWithdrawalAddress(c *cli.Context, r io.Reader) error {
+	ctx, span := trace.StartSpan(c.Context, "withdrawal.blsToExecutionAddress")
+	defer span.End()
+
+	BeaconNodeHost := "http://localhost:3500"
+	if c.String(BeaconHostFlag.Name) != "" {
+		BeaconNodeHost = c.String(BeaconHostFlag.Name)
+	}
+	u, err := url.ParseRequestURI(BeaconNodeHost)
+	if err != nil {
+		return errors.Wrap(err, "invalid format, unable to parse url")
+	}
+	if u.Scheme == "" || u.Host == "" {
+		return fmt.Errorf("url must be in the format of http(s)://host:port url used: %v", BeaconNodeHost)
```

Comment:
```suggestion
		return fmt.Errorf("provided url %s is not in the format of http(s)://host:port", BeaconNodeHost)
```
Error messages should not contain colons because that's how errors are separated.